sw/host/opentitanlib/src/transport/ti50emulator/emu.rs

```diff
@@ -283,6 +295,16 @@ impl EmulatorProcess {
             buffer.resize(prev_len + read_count, 0u8);
             while let Some(newline_pos) = buffer.iter().position(|c| *c == b'\n') {
                 if PATTERN.as_bytes() == &buffer[..newline_pos] {
+                    // Connect to Unix pipes representing each UART.
+                    for weak_rc_uart in &*inner.uarts.borrow() {
+                        match weak_rc_uart.upgrade() {
+                            None => (),
+                            Some(uart) => uart.connect()?,
+                        }
+                    }
+                    // Now release the sub-process to run project code.
+                    child_stdin.write_all("GO\n".as_bytes())?;
```

Comment:
You can probably just write `b"Go\n"` here.

Reply:
Done